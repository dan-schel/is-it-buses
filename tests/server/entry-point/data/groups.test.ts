// TODO: Test that every LineGroup object's constructor ran successfully. Test
// that every station appears at least once in a line group (except for known
// exceptions: the city loop stations & altona loop stations). Write a snapshot
// test for easy validation?
