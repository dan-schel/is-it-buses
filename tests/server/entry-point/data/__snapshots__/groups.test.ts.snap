// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Melbourne line groups > matches the snapshot 1`] = `
"

Ballarat Line
  [Ballarat] Southern Cross → Footscray → Sunshine → Ardeer → Deer Park → Caroline Springs → Rockbank → Cobblebank → Melton → Bacchus Marsh → Ballan → Ballarat → Wendouree → Beaufort → Ararat
  [Ballarat] Southern Cross → Footscray → Sunshine → Ardeer → Deer Park → Caroline Springs → Rockbank → Cobblebank → Melton → Bacchus Marsh → Ballan → Ballarat → Creswick → Clunes → Talbot → Maryborough

Bendigo Line
  [Bendigo] Southern Cross → Footscray → Watergardens → Sunbury → Clarkefield → Riddells Creek → Gisborne → Macedon → Woodend → Kyneton → Malmsbury → Castlemaine → Kangaroo Flat → Bendigo → Eaglehawk → Raywood → Dingee → Pyramid → Kerang → Swan Hill
  [Bendigo] Southern Cross → Footscray → Watergardens → Sunbury → Clarkefield → Riddells Creek → Gisborne → Macedon → Woodend → Kyneton → Malmsbury → Castlemaine → Kangaroo Flat → Bendigo → Epsom → Huntly → Goornong → Elmore → Rochester → Echuca

Belgrave, Lilydale, Alamein, and Glen Waverley lines
  [Belgrave] "The city" → Richmond → East Richmond → Burnley → Hawthorn → Glenferrie → Auburn → Camberwell → East Camberwell → Canterbury → Chatham → Union → Box Hill → Laburnum → Blackburn → Nunawading → Mitcham → Heatherdale → Ringwood → Heathmont → Bayswater → Boronia → Ferntree Gully → Upper Ferntree Gully → Upwey → Tecoma → Belgrave
  [Lilydale] "The city" → Richmond → East Richmond → Burnley → Hawthorn → Glenferrie → Auburn → Camberwell → East Camberwell → Canterbury → Chatham → Union → Box Hill → Laburnum → Blackburn → Nunawading → Mitcham → Heatherdale → Ringwood → Ringwood East → Croydon → Mooroolbark → Lilydale
  [Alamein] "The city" → Richmond → East Richmond → Burnley → Hawthorn → Glenferrie → Auburn → Camberwell → Riversdale → Willison → Hartwell → Burwood → Ashburton → Alamein
  [Glen Waverley] "The city" → Richmond → East Richmond → Burnley → Heyington → Kooyong → Tooronga → Gardiner → Glen Iris → Darling → East Malvern → Holmesglen → Jordanville → Mount Waverley → Syndal → Glen Waverley

Mernda and Hurstbridge lines
  [Mernda] "The city" → Jolimont → West Richmond → North Richmond → Collingwood → Victoria Park → Clifton Hill → Rushall → Merri → Northcote → Croxton → Thornbury → Bell → Preston → Regent → Reservoir → Ruthven → Keon Park → Thomastown → Lalor → Epping → South Morang → Middle Gorge → Hawkstowe → Mernda
  [Hurstbridge] "The city" → Jolimont → West Richmond → North Richmond → Collingwood → Victoria Park → Clifton Hill → Westgarth → Dennis → Fairfield → Alphington → Darebin → Ivanhoe → Eaglemont → Heidelberg → Rosanna → Macleod → Watsonia → Greensborough → Montmorency → Eltham → Diamond Creek → Wattle Glen → Hurstbridge

Pakenham and Cranbourne lines
  [Pakenham] "The city" → Richmond → South Yarra → Caulfield → Carnegie → Murrumbeena → Hughesdale → Oakleigh → Huntingdale → Clayton → Westall → Springvale → Sandown Park → Noble Park → Yarraman → Dandenong → Hallam → Narre Warren → Berwick → Beaconsfield → Officer → Cardinia Road → Pakenham → East Pakenham
  [Cranbourne] "The city" → Richmond → South Yarra → Caulfield → Carnegie → Murrumbeena → Hughesdale → Oakleigh → Huntingdale → Clayton → Westall → Springvale → Sandown Park → Noble Park → Yarraman → Dandenong → Lynbrook → Merinda Park → Cranbourne

Flemington Racecourse Line
  [Flemington Racecourse] Flinders Street → Southern Cross → North Melbourne → Showgrounds → Flemington Racecourse

Frankston Line
  [Frankston] Flinders Street → Richmond → South Yarra → Hawksburn → Toorak → Armadale → Malvern → Caulfield → Glen Huntly → Ormond → McKinnon → Bentleigh → Patterson → Moorabbin → Highett → Southland → Cheltenham → Mentone → Parkdale → Mordialloc → Aspendale → Edithvale → Chelsea → Bonbeach → Carrum → Seaford → Kananook → Frankston

Geelong Line
  [Geelong] Southern Cross → Footscray → Sunshine → Deer Park → Tarneit → Wyndham Vale → Little River → Lara → Corio → North Shore → North Geelong → Geelong → South Geelong → Marshall → Waurn Ponds → Winchelsea → Birregurra → Colac → Camperdown → Terang → Sherwood Park → Warrnambool

Gippsland Line
  [Gippsland] Southern Cross → Flinders Street → Richmond → Caulfield → Clayton → Dandenong → Pakenham → Nar Nar Goon → Tynong → Garfield → Bunyip → Longwarry → Drouin → Warragul → Yarragon → Trafalgar → Moe → Morwell → Traralgon → Rosedale → Sale → Stratford → Bairnsdale

Werribee and Williamstown lines
  [Werribee] Flinders Street → Southern Cross → North Melbourne → South Kensington → Footscray → Seddon → Yarraville → Spotswood → Newport → Laverton → Aircraft → Williams Landing → Hoppers Crossing → Werribee
  [Williamstown] Flinders Street → Southern Cross → North Melbourne → South Kensington → Footscray → Seddon → Yarraville → Spotswood → Newport → North Williamstown → Williamstown Beach → Williamstown

Sunbury, Craigieburn, and Upfield lines
  [Sunbury] "The city" → North Melbourne → Footscray → Middle Footscray → West Footscray → Tottenham → Sunshine → Albion → Ginifer → St Albans → Keilor Plains → Watergardens → Diggers Rest → Sunbury
  [Craigieburn] "The city" → North Melbourne → Kensington → Newmarket → Ascot Vale → Moonee Ponds → Essendon → Glenbervie → Strathmore → Pascoe Vale → Oak Park → Glenroy → Jacana → Broadmeadows → Coolaroo → Roxburgh Park → Craigieburn
  [Upfield] "The city" → North Melbourne → Macaulay → Flemington Bridge → Royal Park → Jewell → Brunswick → Anstey → Moreland → Coburg → Batman → Merlynston → Fawkner → Gowrie → Upfield

Sandringham Line
  [Sandringham] Flinders Street → Richmond → South Yarra → Prahran → Windsor → Balaclava → Ripponlea → Elsternwick → Gardenvale → North Brighton → Middle Brighton → Brighton Beach → Hampton → Sandringham

Seymour Line
  [Seymour] Southern Cross → North Melbourne → Broadmeadows → Craigieburn → Donnybrook → Wallan → Heathcote Junction → Wandong → Kilmore East → Broadford → Tallarook → Seymour → Nagambie → Murchison East → Mooroopna → Shepparton
  [Seymour] Southern Cross → North Melbourne → Broadmeadows → Craigieburn → Donnybrook → Wallan → Heathcote Junction → Wandong → Kilmore East → Broadford → Tallarook → Seymour → Avenel → Euroa → Violet Town → Benalla → Wangaratta → Springhurst → Chiltern → Wodonga → Albury

Stony Point Line
  [Stony Point] Frankston → Leawarra → Baxter → Somerville → Tyabb → Hastings → Bittern → Morradoo → Crib Point → Stony Point

"
`;
